src/js_sync.hpp

```diff
@@ -780,7 +780,7 @@ void SyncClass<T>::populate_sync_config(ContextType ctx, ObjectType realm_constr
     if (Value::is_boolean(ctx, sync_config_value)) {
         config.force_sync_history = Value::to_boolean(ctx, sync_config_value);
         if (config.force_sync_history) {
-            config.schema_mode = SchemaMode::Additive;
+            config.schema_mode = SchemaMode::AdditiveDiscovered;
```

Comment:
JS probably wants AdditiveExplicit. AdditiveDiscovered is for when the schema is auto-discovered from classes so that merely defining an unused embeddedobject subclass doesn't break things.